_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,428 @@
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<div onclick="stop()" aria-hidden="true">
+		<span style="background-color:#333; color:white; padding:5px">Stop updates</span>
+	</div>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+#### Failed Example 4
+
+The text node automatically updates every 3 seconds after the page completes loading. The mechanism to stop the automatic updates has an empty accessible name.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<div onclick="stop()" aria-label="">
+		<span style="background-color:#333; color:white; padding:5px">Stop updates</span>
+	</div>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+#### Failed Example 5
+
+The text node automatically updates every 3 seconds after the page completes loading. The mechanism to stop the automatic updates has an accessible name that is only whitespace.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<div onclick="stop()" aria-label="   ">
+		<span style="background-color:#333; color:white; padding:5px">Stop updates</span>
+	</div>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The text node automatically updates every 3 seconds after the page completes loading but it is not visible.
+
+```html
+<body onload="start()">
+	<p style="display: none">Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+	</script>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The text node automatically updates every 3 seconds but only as a result of an action from the user.
```

Comment:
Is the pause mechanism important for this example? It seems that not auto-starting is enough for 2.2.2 to be happy.